src/video/panoramic.cc

```diff
@@ -80,7 +80,7 @@ std::unique_ptr<Input> getPanoramicCamera()
     }
     // PixPro
     else if(prefCamNum == 1) {
-        return std::make_unique<OpenCVInput>(deviceNum, cv::Size(1440, 1440), "pixpro_usb");
+        return std::make_unique<OpenCVInput>(deviceNum + cv::CAP_V4L, cv::Size(1440, 1440), "pixpro_usb");
```

Comment:
What happens if you don't use ``cv::CAP_V4L``?

Reply:
it uses gstreamer which doesn't SEEM to support setting resolution - this isn't a nice solution though, I might add the a defaulted backend parameter to this function

Reply:
Ah I see. I think this is fine, but it maybe needs a comment in case this workaround doesn't work on someone else's machine.

Reply:
See what you think of my improved solution (could also be useful for convincing @tenxlenx 's mac to work with a USB camera)